lib/generated/lexical/protocol/types/completion/list.ex

```diff
@@ -0,0 +1,30 @@
+# This file's contents are auto-generated. Do not edit.
```

Comment:
Was this generated with the mix task? If not, please generate it with:

`mix lsp.mappings.generate --roots CompletionList` 

I'm asking because I see parens around deftype.

Reply:
good catch